aspnet/hosting/apppool.rst

```diff
@@ -0,0 +1,42 @@
+.. _apppool:
+
+Application Pools
+=================
+
+By `Sourabh Shirhatti`
+
+When hosting multiple Web Sites on a single server, you should consider isolating the applications from each other by running each application in its own application pool. This document provides an overview of how to set up Application Pools to securely host multiple web sites on a single server.
+
+
+Configuring IIS Application Pool Identities
+-------------------------------------------
+
+An application pool identity allows you to run an application under a unique account without having to create and manage domains or local accounts. On IIS 8.0+ the IIS Admin Worker Process (WAS) will create a virtual account with the name of the new application pool and run the application pool's worker processes under this account by default.
+
+In the IIS Management Console, under **Advanced Settings** for your application pool ensure that `Identity` list item is set to use **ApplicationPoolIdentity** as shown in the image below.
+
+.. image:: apppool/_static/apppool-identity.png
+
+Securing Resources
+------------------
+
+THe IIS management process creates a secure identifier with the name of the application pool in the Windows Security System. Resources can be secured by using this identity, however this identity is not a real user account and will not show up in the Windows User Management Console.
+
+To grant the IIS worker process access to your application, you will need to modify the Access Control List (ACL) for the the directory containing your application. 
+
+1. Open Windows Explorer and Naviagate to the directory.
+2. Right click on the directory and click properties.
+3. Under the **Security** tab, click the **Edit** button and then the **Add** button
+4. Click the **Locations** and make sure you select your server.
+
```

Comment:
@shirhatti ... but are you saying that you **did** need to explicitly set permissions, or are you saying you **didn't** set them and that that section pertains to setting permissions for some other purpose. I was pointing out that everything up to this statement ...
```
To grant the IIS worker process access to your application
```
is all good. But I get confused at that point. It makes it sound like the dev needs to configure permissions for the root folder itself ... for the application itself, but that shouldn't be necessary. I'm wondering if what you're talking about here are *other* folders outside of the root dir on the server or maybe something for virtual directories.

Let me ask you the other way around: Have you had trouble (e.g., 502.3 error) when you **don't** grant explicit permissions for any account to the root folder of an app? I mean, is the pass-through auth with the default AppPool identity failing?

btw- I'm only aware from one quick test I did with pass-through auth and a fileshare that it failed even with explicit permissions being set, but I think that was the old vdir problem ... maybe that's been fixed ... not sure.

But coming back to my main point: It just seems a little confusing here, because the text up to ...
```
To grant the IIS worker process access to your application, 
```
sounds great, then it looks like the doc is telling devs that permissions must be set even for a default app setup.

Certainly, I might be missing something here. So just throw a :wrench: at me if I have missed the point or scenario this addresses. :smile: